packages/flutter_test/lib/src/controller.dart

```diff
@@ -991,6 +1019,48 @@ abstract class WidgetController {
     return tapAt(getCenter(finder, warnIfMissed: warnIfMissed, callee: 'tap'), pointer: pointer, buttons: buttons);
   }
 
+  /// Dispatch a pointer down / pointer up sequence at the first hit-testable
+  /// [InlineSpan] (typically a [TextSpan]) within the given text range.
+  ///
+  /// This method performs a more spatially precise tap action on a piece of
+  /// static text, than the widget-based [tap] method.
+  ///
+  /// The given [Finder] must find one and only one matching substring, and the
+  /// substring must be hit-testable (meaning, it must not be off-screen, or be
+  /// obscured by other widgets, or in a disabled widget). Otherwise this method
+  /// throws a [FlutterError].
+  ///
+  /// If the target substring contains more than one hit-testable [InlineSpan]s,
+  /// [tapOnText] taps on one of them, but does not guarantee which.
+  ///
+  /// This method currently only works on static text widgets ([Text] or
+  /// [RichText] for example) that use [RenderParagraph] under the hood. It
+  /// currently does not support finding substrings in [TextField]s or
```

Comment:
Remove the "currently"s in these two sentences? 

"does not support" => will it throw if it find the substring in TextFields? Will it just not see that text at all? Maybe be a little more specific what "not supported" means.